WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m

```diff
@@ -123,13 +122,6 @@ - (void)viewWillAppear:(BOOL)animated
 {
     [super viewWillAppear:animated];
 
-    if (self.previousOrientation != UIInterfaceOrientationUnknown) {
-        if (UIInterfaceOrientationIsPortrait(self.previousOrientation) != UIInterfaceOrientationIsPortrait(self.interfaceOrientation)) {
-            [self.tableViewHandler refreshCachedRowHeightsForWidth:CGRectGetWidth(self.view.frame)];
```

Comment:
Nice catch @jleandroperez.  
Yeah the idea here was the user might rotate the device while viewing a link or video.  When they return, layout would need to be refreshed.